frontend/pages/organizations/[id]/events.vue

```diff
@@ -27,7 +27,10 @@
         />
       </div>
     </HeaderAppPage>
-    <div v-if="organization.events?.length > 0" class="space-y-3 py-4">
+    <div
+      v-if="organization.events && organization.events.length > 0"
```

Comment:
Was what I was thinking too, but also weird that it's not needed for groups 🤔

Reply:
Mhm, I'll take a look at the groups later after work.

Reply:
Oh, it's because the `Events` type was previously under the `dom-events` type by default, but since there is no default type for groups, it's defined as an `any`. The `any` type wouldn't have any problem under this case.